### ISSUE TITLE
Shading on FBX from blender 4.1 is broken. 

### ISSUE THREAD

**Issue by Muzz** [FlaxEngine/FlaxEngine#2397]
**Issue description:**
I migrated over to blender 4.1 today. But for some reason the normals of models exported from blender 4.1 and imported to flax are totally broken. 

![image](https://github.com/FlaxEngine/FlaxEngine/assets/2074204/68be08f9-a8b6-42c1-a1e8-7fad8b7b57db)

I suspect it's a newer version of FBX, but that's based on a guess. 
![image-1](https://github.com/FlaxEngine/FlaxEngine/assets/2074204/c2de650c-f39a-438e-9c3c-bc36466b46e9)

how it should look

**Steps to reproduce:**
Open blender 4.1 make a sphere and smooth it, export to FBX and import to flax. 

**Minimal reproduction project:**

[shadeTest.zip](https://github.com/FlaxEngine/FlaxEngine/files/14883623/shadeTest.zip)

Master

**Comment by mafiesto4** [comment]
I have no idea how to properly import those enormals in OpenFBX. Probably using Autodesk FBX SDK would help. I hope someone else has an idea. Thx for adding sample mesh, this helps testing.

I tested Unity 2018 and it throws:
```
ImportFBX Errors:
The mesh Cube has invalid normals. Try cleaning and triangulating Cube in your 3D modeller before importing it in unity.
This is a failure in the fbx exporter of the tool which exported the fbx file.
```

**Comment by mafiesto4** [comment]
Fixed in #2672
